src/plots.jl

```diff
@@ -0,0 +1,151 @@
+# Convert coordinates to the form used by Plots.jl
+_coordvecs(::PointTrait, geom) = [tuple(coordinates(geom)...)]
```

Comment:
Can we put all these functions in the extension and just leave the stubs here to extend in the extension?